utils/helpers.js

```diff
@@ -267,6 +269,19 @@ const unPauseAndPoll = async(ERC20Pausable) => {
 
     return false // paused
 }
+
+const pollForEvent = async(contract) => {
```

Comment:
https://github.com/hashgraph/hedera-smart-contracts/pull/584/files#diff-ab063aef47051b3442202d7087747109a96c12399fd2dd7c0c92a48f2c4c32a2R271 passes 2 params.
I prefer that so we should utilize that param here